internal/config/cardpick.go

```diff
@@ -0,0 +1,83 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/vinegarhq/vinegar/sysinfo"
+)
+
+// opt accepts the following values:
+// aliases - "integrated", "prime-discrete" and "none": Equivalent to "0", "1" or empty. Enables an extra "prime" check.
+// integer - GPU index
+// empty   - Skips logic and does nothing.
+func (b *Binary) pickCard() error {
+	opt := b.ForcedGpu
```

Comment:
Either make this a pointer or just use it directly below instead of this

Reply:
Fixed.